static/js/lib/courses.js

```diff
@@ -24,3 +24,8 @@ export const maxPrice = (course: Course) => {
   const price = Math.max(...R.map(R.view(R.lensProp("price")), course.prices))
   return price > 0 ? `$${price}` : "Free"
 }
+
+export const minPrice = (course: Course) => {
+  const price = Math.min(...R.map(R.view(R.lensProp("price")), course.prices))
```

Comment:
could this just be `Math.min(...course.prices.map(price => price.price))` or `Math.min(...course.prices.map(R.prop('price')))` ? might be just a little bit more succinct, unless there's some reason we need to use the lens

Reply:
Thanks, done